feedstock/catalog.yaml

```
@@ -14,50 +14,50 @@ tags:
 stores:
   - id: "GLEAM"
     name: "GLEAM version 4 Global Evapotranspiration"
-    url: "https://nyu1.osn.mghpcc.org/leap-pangeo-pipeline/GLEAM/GLEAM.zarr"
+    url: "gs://leap-persistent/data-library/GLEAM/GLEAM-{variable}.zarr"
```

```suggestion
    url: "gs://leap-persistent/data-library/GLEAM/GLEAM.zarr""
```

This should be a single Zarr store containing multiple vars.